### PR TITLE
feat: add in progress swap history badge and action required notification

### PR DESCRIPTION
**Pull request by RyukTheCoder**
# Summary

added in progress swap history badge to home page and added action required state to unread notifications badge

Fixes # RF-1240

# How did you test this change?

Tested widget on Chrome browser.

# Checklist:

- [x] I have performed a self-review of my code
- [x] Implemented a user interface (UI) change, referencing our Figma design to ensure pixel-perfect precision.

### CODE REVIEW

**Review by samobasquiat — CHANGES_REQUESTED** [review]

**Comment by samobasquiat** [comment]
I think it's better to handle the hover state in this PR as well.  What do you think?

File: widget/ui/src/icons/InProgress.tsx [hunk 1]
```diff
@@ -8,24 +8,34 @@ function SvgInProgress(props: SvgIconPropsWithChildren) {
   return createElement(
```
[the remark this answers is not in the document]
Comment (RyukTheCoder) [reply widget/ui/src/icons/InProgress.tsx:8]: The icon already existed but has been changed in design.

**Comment by RyukTheCoder** [comment]
> I think it's better to handle the hover state in this PR as well. What do you think?

Added hover state to header buttons. Check please @samobasquiat

File: widget/ui/src/icons/InProgress.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (samobasquiat) [reply widget/ui/src/icons/InProgress.tsx:8]: Yeah, but I meant the resource should change as well.

**Comment by samobasquiat** [comment]
> > I think it's better to handle the hover state in this PR as well. What do you think?
> 
> Added hover state to header buttons. Check please @samobasquiat

image
image
image

File: widget/ui/src/icons/InProgress.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (RyukTheCoder) [reply widget/ui/src/icons/InProgress.tsx:8]: Thanks, fixed

**Comment by RyukTheCoder** [comment]
> > > I think it's better to handle the hover state in this PR as well. What do you think?
> > 
> > 
> > Added hover state to header buttons. Check please @samobasquiat
> 
> image image image

Done

**Comment by samobasquiat** [comment]
image
image
I think we can add a minor padding here (or another way) to divide them.

**Comment by RyukTheCoder** [comment]
> image image I think we can add a minor padding here (or another way) to separate them.

Done using Divider component

**Review by samobasquiat — APPROVED** [review]

File: widget/embedded/src/components/HeaderButtons/InProgressTransactionBadge.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import type { PendingSwap } from 'rango-types';
+
+import { useManager } from '@rango-dev/queue-manager-react';
+import { InProgressIcon, styled } from '@rango-dev/ui';
+import React from 'react';
+
+import { getPendingSwaps } from '../../utils/queue';
+
+export const Container = styled('div', {
```
Comment (nikaaru) [inline widget/embedded/src/components/HeaderButtons/InProgressTransactionBadge.tsx:9]: I think it's better to have all the styles in the **HeaderButtons.styles.ts**

Comment (RyukTheCoder) [reply widget/embedded/src/components/HeaderButtons/InProgressTransactionBadge.tsx]: fixed

**Review by nikaaru — APPROVED** [review]

**Pull request merged**